test/e2e/test/agent/builder.go

```diff
@@ -192,7 +192,7 @@ func (b Builder) WithFleetAgentDataStreamsValidation() Builder {
 		WithDefaultESValidation(HasWorkingDataStream(MetricsType, "elastic_agent.fleet_server", "default")).
 		WithDefaultESValidation(HasWorkingDataStream(MetricsType, "elastic_agent.metricbeat", "default"))
 	// https://github.com/elastic/cloud-on-k8s/issues/7389
-	if v.LT(version.MinFor(8, 12, 0)) || v.GE(version.MinFor(8, 14, 0)) {
+	if v.LT(version.MinFor(8, 12, 0)) || v.GE(version.MinFor(8, 15, 0)) {
```

Comment:
I'm sorry, I could be sleep deprived, but where is 8.15.x coming from? From reading through #7389, shouldn't we just say:

```
if v.LT(version.MinFor(8, 12, 0)) {
```

Isn't it going to be this way going forward? We don't want to have to keep bumping the `GE` version, right?

Reply:
That is a good question, I had the same question as well. But I left it in so we could discuss in this PR. I am not sure  why we have an upper bound on this.